Lease Search Results Component - Rollup Build Type Error

```
[!] (plugin rpt2) Error: /home/runner/work/ui-components/ui-components/src/components/LeaseSearchResults.tsx(22,25): semantic error TS2339: Property 'image' does not exist on type 'Object'.
src/components/LeaseSearchResults.tsx
Error: /home/runner/work/ui-components/ui-components/src/components/LeaseSearchResults.tsx(22,25): semantic error TS2339: Property 'image' does not exist on type 'Object'.
```

Package will not deploy until the type definitions are locked down.

--- a/.gitignore
+++ b/.gitignore
@@ -9,8 +9,8 @@
 /coverage
 
 # production
-/build
-/dist
+/build/
+/dist/
 
 # misc
 .DS_Store

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
     "@babel/preset-react": "^7.10.1",
     "@babel/preset-typescript": "^7.10.1",
     "@material-ui/core": "^4.10.1",
+    "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
@@ -32,6 +33,7 @@
     "rollup-plugin-commonjs": "^10.1.0",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-peer-deps-external": "^2.2.2",
+    "rollup-plugin-postcss": "^3.1.2",
     "rollup-plugin-typescript2": "^0.27.1",
     "styled-components": "^5.1.1",
     "tailwindcss": "^1.4.6",
@@ -44,7 +46,7 @@
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "build-storybook -s public",
-    "build:css": "npx tailwindcss build ./src/tailwindcss/tailwind.css -o ./src/tailwindcss/App.css"
+    "build-css": "npx tailwindcss build ./src/tailwindcss/tailwind.css -o ./src/tailwindcss/App.css"
   },
   "eslintConfig": {
     "extends": "react-app"

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -3,6 +3,8 @@ import babel from "rollup-plugin-babel";
 import resolve from "rollup-plugin-node-resolve";
 import commonjs from 'rollup-plugin-commonjs';
 import typescript from 'rollup-plugin-typescript2';
+import postcss from "rollup-plugin-postcss";
+
 import pkg from './package.json'
 
 const rootDir = path.resolve(__dirname);
@@ -20,7 +22,10 @@ export default {
     }
   ],
   plugins: [
-    typescript({ objectHashIgnoreUnknownHack: true })
+    typescript({ objectHashIgnoreUnknownHack: true }),
+    postcss({
+      extensions: [ '.css' ],
+    })
   ],
   external: ['react', 'react-dom']
 }

--- a/src/components/LeaseSearchBar.tsx
+++ b/src/components/LeaseSearchBar.tsx
@@ -1,5 +1,4 @@
 import React from "react";
-import { ReactComponent as Search } from "../assets/search.svg";
 
 import {
   TextField,
@@ -9,6 +8,8 @@ import {
   FormLabel,
   Grid,
 } from "@material-ui/core";
+
+import SearchIcon from '@material-ui/icons/Search';
 
 const useStyles = makeStyles({
   bed: {
@@ -64,7 +65,7 @@ export const LeaseSearchBar = () => {
                 }}
               >
                 <span>
-                  <Search width="36px" height="36px" />
+                  <SearchIcon />
                 </span>
               </button>
             </Grid>

--- a/src/components/LeaseSearchResults.tsx
+++ b/src/components/LeaseSearchResults.tsx
@@ -1,8 +1,17 @@
 import React, { FunctionComponent } from "react";
 import "../tailwindcss/App.css";
 
+type ListObj = {
+  image: string;
+  address: string;
+  contract: string;
+  BIN: Number;
+  rent: Number;
+  bedRooms: Number;
+}
+
 type SearchProps = {
-  data: Array<Object>;
+  data: Array<ListObj>;
 };
 
 const LeaseSearchResults: FunctionComponent<SearchProps> = ({ data }) => {
@@ -20,7 +29,7 @@ const LeaseSearchResults: FunctionComponent<SearchProps> = ({ data }) => {
           <div className="flex my-4 bg-gray-100 px-4 py-4 justify-between">
             <img
               src={item.image}
-              alt=""
+              alt={item.address}
               className="w-32 h-32 object-cover rounded"
             />
             <div className="bg-white flex justify-between w-full mx-4 py-6 px-4 rounded">

--- a/src/stories/3-LeaseResults.stories.tsx
+++ b/src/stories/3-LeaseResults.stories.tsx
@@ -10,7 +10,7 @@ const mockData = [
   {
     address: "411 Washington",
     contract: "49184027148s29j",
-    rent: "500",
+    rent: 500,
     bedRooms: 3,
     BIN: 123,
     image: require("../assets/realestate.jpeg"),
@@ -18,7 +18,7 @@ const mockData = [
   {
     address: "402 Washington",
     contract: "1340589238jx0",
-    rent: "300",
+    rent: 300,
     bedRooms: 2,
     BIN: 432,
     image: require("../assets/realestate1.jpg"),

